TaskManager/ViewController.swift

```diff
@@ -0,0 +1,73 @@
+//
+//  ViewController.swift
+//  TaskManager
+//
+//  Created by user on 14.09.2022.
+//
+
+import UIKit
+
+class ViewController: UIViewController {
```

Comment:
lets use MVP architecture instead, please check the video I shared, and if you dont understand something let me know